src/overlays/actors/ovl_En_Hata/z_en_hata.c

```diff
@@ -28,16 +29,74 @@ const ActorInit En_Hata_InitVars = {
     (ActorFunc)EnHata_Draw,
 };
 
-#endif
+void EnHata_Init(Actor* thisx, GlobalContext* globalCtx) {
+    s32 rand;
+    EnHata* this = THIS;
+    f32 endFrame;
+
+    SkelAnime_Init(globalCtx, &this->skelAnime, &object_hata_Skel_002FD0, NULL, this->jointTable, this->morphTable,
+                   OBJECT_HATA_LIMB_MAX);
+    endFrame = Animation_GetLastFrame(&object_hata_Anim_000444);
+    Animation_Change(&this->skelAnime, &object_hata_Anim_000444, 1.0f, 0.0f, endFrame, 0, 0.0f);
+    rand = Rand_ZeroFloat(endFrame);
+    this->skelAnime.curFrame = rand;
+    DynaPolyActor_LoadMesh(globalCtx, &this->dyna, &object_hata_Colheader_0000C0);
+    Actor_SetScale(&this->dyna.actor, 0.013f);
+    this->dyna.actor.uncullZoneScale = 500.0f;
+    this->dyna.actor.uncullZoneDownward = 500.0f;
+    this->dyna.actor.uncullZoneForward = 2200.0f;
+}
+
+void EnHata_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+    EnHata* this = THIS;
+
+    DynaPoly_DeleteBgActor(globalCtx, &globalCtx->colCtx.dyna, this->dyna.bgId);
+}
+
+void EnHata_Update(Actor* thisx, GlobalContext* globalCtx2) {
+    GlobalContext* globalCtx = globalCtx2;
+    EnHata* this = THIS;
+    Vec3f sp34;
+    f32 phi_fv0;
+    f32 phi_fv0_3;
```

Comment:
Is this used?

Reply:
no,  but it's needed for padding, i renamed it to s32 pad.